### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Vova-Stoliar**
[DEMO LINK](https://Vova-Stoliar.github.io/react_movies-list/)

### CODE REVIEW

**Review by george-w-boo — APPROVED** [review]
В принципі все норм, за виключенням того, що demo-link не працює та не виконана 4-та умова задачі: All the fields in the movie except the description are required

File: src/components/MoviesList/MoviesList.tsx [hunk 1]
```diff
@@ -1,6 +1,34 @@
 import React from 'react';
-import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Film {
+  [key: string]: string;
+}
```
Comment (varseniuk) [inline src/components/MoviesList/MoviesList.tsx:6]:
```suggestion
interface Film {
  [key: string]: string;
}
```
Let's describe all fields of this object (title. description, imdbId etc.) and move this interface to a separate file (like `src/types/Film.tsx`)

File: src/components/MovieCard/MovieCard.tsx [hunk 2]
```diff
@@ -2,6 +2,48 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Film = {
+  [key: string]: string;
+};
```
Comment (varseniuk) [inline src/components/MovieCard/MovieCard.tsx:7]:
```suggestion
type Props = {
  film: Film;
};
```
You should describe props here

File: src/components/MovieCard/MovieCard.tsx [hunk 3]
```diff
@@ -2,6 +2,48 @@ import React from 'react';
...
+
+export const MovieCard: React.FC<Film> = (film) => {
+  const {
+    title, description, imgUrl, imdbUrl,
+  } = film;
```
Comment (varseniuk) [inline src/components/MovieCard/MovieCard.tsx:12]:
```suggestion
export const MovieCard: React.FC<Props> = (props) => {
  const {
    title, description, imgUrl, imdbUrl,
  } = props;
```

File: src/components/MoviesList/MoviesList.tsx [hunk 4]
```diff
@@ -1,6 +1,34 @@
...
+
+type Props = {
+  films: Film[];
+};
+
+export const MoviesList: React.FC<Props> = ({ films }: Props) => (
```
Comment (varseniuk) [inline src/components/MoviesList/MoviesList.tsx:12]:
```suggestion
  films: Film[];
};

export const MoviesList: React.FC<Props> = (props) => {
  const { films } = props;
```

File: src/components/MoviesList/MoviesList.tsx [hunk 5]
```diff
@@ -1,6 +1,34 @@
...
+  <>
```
Comment (varseniuk) [inline src/components/MoviesList/MoviesList.tsx:13]:
```suggestion
```
Why do you need this fragment here? This component already returns only one element - div

**Review by varseniuk — CHANGES_REQUESTED** [review]

**Review by varseniuk — APPROVED** [review]
Good job!

**Pull request closed without merging**